### ISSUE TITLE
Review-2019.7.8-赵启浩

### ISSUE THREAD

**Issue by fistyee** [DIYer22/weekly-review#575]
**Review**

- 完成了腾讯在线文档中的pytorch 60min教程与python可视化方法

- 学习深度强化学习

- 阅读《SeedNet: Automatic Seed Generation with Deep Reinforcement Learning
for Robust Interactive Segmentation》 
      概述:交互式分各中，自动生成前景与背景点
      过程:使用Random Walk分割算法作为现成的交互分割算法。用户先输入一个前景与背景点，通过这个RW算法得到的分割二值图和原图像一起作为DQN网络的输入，然后DQN网络输出新的种子(前景/背景点)信息，新的种子信息包括位置和标签（属于前景/背景）。最后，新产生的种子点加入并更新原先的种子图。此外，再把更新完的种子信息和原图像放在一起，用RW分割算法求出新的二值图。新的二值图一方面与GT mask进行损失计算得到Reward值，另一方面需要和原图像一起作为DQN新的输入。

**Next**

- py完成全连接神经网络并拟合sin曲线

- 学习深度强化学习课程整理论文，学习pytorch的DQN[教程](https://pytorch.org/tutorials/intermediate/reinforcement_q_learning.html)

- 阅读[《Interactive Image Segmentation via Backpropagating Refinement Scheme》](http://openaccess.thecvf.com/content_CVPR_2019/papers/Jang_Interactive_Image_Segmentation_via_Backpropagating_Refinement_Scheme_CVPR_2019_paper.pdf)
